cg/meta/upload/raredisease/raredisease.py

```diff
@@ -31,8 +31,12 @@ def upload(self, ctx: click.Context, case: Case, restart: bool) -> None:
         """Uploads RAREDISEASE analysis data and files."""
         analysis: Analysis = case.analyses[0]
         self.update_upload_started_at(analysis=analysis)
+
         ctx.invoke(upload_observations_to_loqusdb, case_id=case.internal_id)
 
+        # Clinical delivery upload
+        ctx.invoke(upload_clinical_delivery, case_id=case.internal_id)
```

Comment:
should we add the delivery report generation to the uploads as well?

Reply:
I can add it with the last command (genotype) and make the big test then?

Reply:
sounds good 👍